src/Microsoft.VisualBasic/src/Microsoft.VisualBasic.builds

```diff
@@ -2,7 +2,8 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="Microsoft.VisualBasic.vbproj" />
+    <!-- Temporary workaround, VisualBasic projects cannot be built on non-Windows platforms. See issue #5230. -->
+    <Project Include="Microsoft.VisualBasic.vbproj" Condition="'$(OSEnvironment)' == 'Windows_NT'" />
```

Comment:
I suspect you did it this way so that it could still build for other OSGroups on windows in which case I'm fine with this as a temporary fix.